api_app/db/repositories/airlock_requests.py

```diff
@@ -18,6 +20,10 @@ class AirlockRequestRepository(AirlockResourceRepository):
     def __init__(self, client: CosmosClient):
         super().__init__(client)
 
+    @staticmethod
+    def airlock_requests_query():
+        return 'SELECT * FROM c WHERE'
```

Comment:
Is this working when there are reviews as well? In that collection in cosmos two types are saved: airlock-request and airlock-review. It seems it will return both

Reply:
Thank you ever so much!!! :-)